WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -210,6 +206,9 @@ import WordPressComAnalytics
         // pull to refresh animation.
         view.userInteractionEnabled = readerTopic != nil
 
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(defaultAccountDidChange(_:)), name: WPAccountDefaultWordPressComAccountChangedNotification, object: nil)
```

Comment:
If I remember correctly, the splitview implementation may completely tear down and recreate the splitviews on account change anyway.